substratools/function.py

```diff
@@ -264,12 +178,13 @@ def load_performance(path: os.PathLike) -> Any:
     return performance
 
 
-def execute(interface, sysargs=None):
-    """Launch algo command line interface."""
+def execute(functions_list, sysargs=None):
```

Comment:
should be *args (IMO) and get a tuples of function (static element, better for memory consumption)